### DISCUSSION TITLE
Wallet Connect connection not working

### DISCUSSION THREAD

**Discussion by GouthamJM** [WalletConnect/walletconnect-monorepo#2697] [Developer Support] [answered]
- We have implemented wallet connect v2 , in the dapp https://web-v2-git-feat-walletconnectv2-frontier.vercel.app/portfolio
- But I am not able to make a wallet connection to my dapp , I tried with trust wallet , zerion and frontier android and ios apps, none of them seem to work 
- Even with this official example https://react-app.walletconnect.com/ , it is not working .
- my relay url is - wss://relay.walletconnect.com

**Comment by shan-c** [comment]
same here

**Comment by GouthamJM** [comment, marked as answer]
The issue is fixed - it is working fine now , issue was "Partial Relay Outage in the Southeast Asia Region"

**Comment by joma1230** [comment]
dont work((

**Comment by vikserov14** [comment]
everything works
